modules/mod_ginger_edit/templates/_admin_tinymce_overrides_js.tpl

```diff
@@ -1,7 +1,19 @@
 /*
 Custom settings to override tiny-init.js.
 */
-tinyInit.language="{{ m.translation.language|default:"en" }}";
-tinyInit.menubar="";
-tinyInit.toolbar="styleselect | bold italic | bullist numlist | removeformat | zlink zmedia | link unlink | code";
-tinyInit.extended_valid_elements="iframe[src|style|width|height|scrolling|marginwidth|marginheight|frameborder]";
+
+if (typeof tinymce_overrides_language === 'undefined' || !tinymce_overrides_language) {
+    tinyInit.language="{{ m.translation.language|default:"en" }}";
+}
+
+if (typeof tinymce_overrides_menubar === 'undefined' || !tinymce_overrides_menubar) {
+    tinyInit.menubar="";
+}
+
+if (typeof tinymce_overrides_toolbar === 'undefined' || !tinymce_overrides_toolbar) {
+    tinyInit.toolbar="styleselect | bold italic | bullist numlist | removeformat | zlink zmedia | link unlink | code";
+}
+
+if (typeof tinymce_overrides_extended_valid_elements === 'undefined' || !tinymce_overrides_extended_valid_elements) {
+    tinyInit.extended_valid_elements="iframe[src|style|width|height|scrolling|marginwidth|marginheight|frameborder]";
```

Comment:
We could do this, but then it would not be possible to remove an element from the allowed list. We could use a parameter to determine if it should overwrite or add?